iron-overlay-behavior.html

```diff
@@ -183,10 +202,12 @@
     detached: function() {
       Polymer.dom(this).unobserveNodes(this._observer);
       this._observer = null;
-      if (this.__raf) {
-        window.cancelAnimationFrame(this.__raf);
-        this.__raf = null;
+      for (var cb in this.__rafs) {
+        if (this.__rafs[cb]) {
```

Comment:
I have a personal style preference for checking more explicitly in cases like this e.g., `this.__rafs[cb] != null` or whatever is most appropriate for the condition.